mwdb/resources/api_key.py

```diff
@@ -64,8 +74,12 @@ def post(self, login):
         except NoResultFound:
             raise NotFound("User not found")
 
+        data = request.get_data(as_text=True) or "{}"
+        key_name = loads_schema(data, APIKeyIssueRequestSchema())["name"]
```

Comment:
It would be nice to add specification of this new field into OpenAPI specification (see `post()` method docstring)

Search for `requestBody` in other resources for more information.